### DISCUSSION TITLE
How to load coverart with filenames like cover1, cover2?

### DISCUSSION THREAD

**Discussion by rivke41levp656** [mpv-player/mpv#14520] [Q&A] [answered]
It seems only "^cover$".ext is in the `cover-art-whitelist`. 
I want to load any ^cover.*.ext without setting `cover-art-auto=all`.

**Comment by guidocella** [comment, marked as answer]
https://github.com/mpv-player/mpv/pull/14533 will let you specify `--cover-art-whitelist-add=cover1,cover2,cover3`

Is that enough to fix the issue?

**Reply by rivke41levp656** [reply]
Yes. That looks good. Thank you.
